tasks/gen/model.cr

```diff
@@ -3,8 +3,10 @@ require "teeplate"
 require "avram"
 require "./templates/model_template"
 require "wordsmith"
+require "./mixins/migration_with_columns"
 
 class Gen::Model < LuckyCli::Task
+  include Gen::Mixins::MigrationWithColumns
```

Comment:
Tiny thing, but could you add a newline after this? We typically put newlines after includes

Reply:
I did initially because I love the space, but then I removed it. I will add one again.